docs/runbooks/HPPSharingPoolPathWithOS.md

```diff
@@ -1,34 +1,51 @@
+<!-- Edited by Jiří Herrmann, 8 Nov 2022 -->
+
 # HPPSharingPoolPathWithOS
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-If HPP is sharing a filesystem with other critical components (kubelet / OS) its possible that HPP PVs will cause node disk pressure.
+This alert fires when the hostpath provisioner (HPP) is sharing a file system with other critical components, such as `kubelet` or the operating system (OS).
+
+HPP dynamically provisions hostpath volumes to provide storage for persistent volume claims (PVCs). In the described scenario, HPP persistent volumes might cause node disk pressure.
```

Comment:
You can remove the 2nd sentence because it describes the impact rather than meaning.